lib/resources/r4/related_person_patch.yaml

```diff
@@ -64,3 +64,25 @@ operations:
         <li>Must include a time component with a timezone when specifying a `period`.</li>
         <li>If `line` is provided, only the first four values are used. Additional values will be ignored.</li>
       </ul>
+
+  - name: replace-name-given
+    path: /name/{index}/given
+    operation: replace
+    type: String
+    url: https://hl7.org/fhir/R4/datatypes-definitions.html#HumanName.given
+    description: Replace the `HumanName.given` of the name at the given <code>{index}</code> in the list of related person names.
+    example: |
+      {
+        "path": "/name/0/given",
+        "op": "replace",
+        "value": [
+          "Harry",
+          "James"
+        ]
+      }
+    note: >
+      <ul>
+        <li>Requires a <a href="#test-name-id">test operation</a> to be provided for the `HumanName.id` of the name whose `given` is intended to be replaced.</li>
+        <li>This operation is only supported for the first name in the list of related person names (<code>{index}</code> == 0).</li>
+        <li>If fewer values for given are provided in the list than previously set, the additional older values will be unset.</li>
+      </ul>
```

Comment:
missing newline at the end

Reply:
added: https://github.com/cerner/fhir.cerner.com/commit/0adab3f661325249f23031c7415eae48d2bc9929#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR123

Reply:
Added : https://github.com/cerner/fhir.cerner.com/commit/0adab3f661325249f23031c7415eae48d2bc9929#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR123